server/models/NonProfit.js

```diff
@@ -20,8 +17,8 @@ const nonProfitSchema = new Schema({
     type: String,
     required: true,
   }
-
 });
 
+const NonProfit = mongoose.model('NonProfit', nonProfitSchema);
 
-module.exports = nonProfitSchema;
```

Comment:
keep - module.exports = nonProfitSchema
and remove - module.exports = NonProfit;